tests/test_preprocessing.py

```diff
@@ -255,13 +255,13 @@ def check_scaling_features(use_feature_hashing=False, use_scaling=False):
 
     # these are the expected values of the f-measures, sorted
     if not use_feature_hashing:
-        expected_fmeasures = ([0.77319587628865982, 0.78640776699029125] if
+        expected_fmeasures = ([0.55276381909547745, 0.55721393034825872] if
                               not use_scaling else
-                              [0.94930875576036866, 0.93989071038251359])
+                              [0.65217391304347827, 0.70370370370370372])
```

Comment:
Yeah. I was wondering about this myself. The changes in the other tests aren't as drastic but I am not sure what we can do about this.  Note also that the numbers look more reasonable to me now both with and without scaling and the direction is what we'd expect (scaling leads to better performance). Perhaps the previous implementation had bugs? 

@mheilman any thoughts?